# Sparse matrix data

int64 rows
int64 cols
float64 cellSize
geometry_msgs/Point origin
TripletInt[] elements
